src/ot_orchestration/dags/config/genetics_etl.yaml

```
@@ -208,6 +209,7 @@ nodes:
       step.credible_set_path: gs://ot_orchestration/releases/24.10_freeze6/credible_set
       step.download_from_hub: true
       step.hf_hub_repo_id: opentargets/locus_to_gene
```

This should be changed if you want this config to be an accurate representation of the dag, right?
```suggestion
      step.hf_hub_repo_id: opentargets/locus_to_gene_egl
```